homeassistant/components/cover/mqtt.py

```diff
@@ -190,3 +254,35 @@ def async_stop_cover(self, **kwargs):
         mqtt.async_publish(
             self.hass, self._command_topic, self._payload_stop, self._qos,
             self._retain)
+
+    @asyncio.coroutine
+    def async_open_cover_tilt(self, **kwargs):
+        """Tilt the cover open."""
+        mqtt.async_publish(self.hass, self._tilt_command_topic,
+                           self._tilt_open_position, self._qos, self._retain)
+
+    @asyncio.coroutine
+    def async_close_cover_tilt(self, **kwargs):
+        """Tilt the cover closed."""
+        mqtt.async_publish(self.hass, self._tilt_command_topic,
+                           self._tilt_closed_position, self._qos, self._retain)
+
+    @property
+    def current_cover_tilt_position(self):
+        """Return current position of cover tilt."""
+        return self._tilt_value
+
+    @asyncio.coroutine
+    def async_set_cover_tilt_position(self, **kwargs):
+        """Move the cover tilt to a specific position."""
+        if ATTR_TILT_POSITION in kwargs:
```

Comment:
Make this a guard clause instead, as shown below. We prefer this pattern because it helps keep the indentation manageable in larger methods. Not a huge deal here, but it's better to be consistent.

```python
if ATTR_TILT_POSITION not in kwargs:
    return
```